### PR TITLE
Validations and model specs

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
# Add validations and unit testing

### For user :
- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.

https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/84406aa85211cd74ad966f9c32c41ff9f6ba020f/app/models/user.rb#L6-L8

### For post:
- The title must not be blank.
- The title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/84406aa85211cd74ad966f9c32c41ff9f6ba020f/app/models/post.rb#L6-L9

## Add unit test for all the validations and  methods on the models:
- User:
https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/e9b2e67ab7c916d9ff79a409806967dafce090b5/spec/models/user_spec.rb#L1-L35

- Post:
https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/e9b2e67ab7c916d9ff79a409806967dafce090b5/spec/models/post_spec.rb#L1-L50

- Comment:
https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/e9b2e67ab7c916d9ff79a409806967dafce090b5/spec/models/comment_spec.rb#L1-L12

- Like:
https://github.com/Alejandro-Bernal-M/rails-blog-app/blob/e9b2e67ab7c916d9ff79a409806967dafce090b5/spec/models/like_spec.rb#L1-L12

image

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  # Validations
+  validates :Title, presence: true, length: { minimun: 3, maximum: 250 }
```
Comment (fobadara) [inline app/models/post.rb]: [REQUIRED] There is a mistake in your code. You typed `minimun`  instead of `minimum`

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Alejandro-Bernal-M, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good job including tests for your models

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/post.rb (hunk 1 above)
Comment (Alejandro-Bernal-M) [reply app/models/post.rb]: Thank you! I did not notice that, I will correct it.

**Review by victorokoroji — APPROVED** [review]
Hi @Alejandro-Bernal-M , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Required changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**